src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+$hand-transform-origin: center bottom;
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin starting-position {
+  position: absolute;
+  bottom: 50%;
+}
+
 body {
+  @include center;
+
   margin: 0;
+  height: 100vh;
+}
+
+@keyframes infinite-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
```

Comment:
It is advised to use the shorthand `0` instead of `0deg` for the `rotate` value in the `transform` property.